api/src/main/java/org/openmrs/module/appointments/service/impl/RecurringAppointmentServiceImpl.java

```diff
@@ -127,13 +160,14 @@ private Date getStartTime(Appointment pendingAppointment, Appointment appointmen
         return getUpdatedTimeStamp(startHours, startMinutes, pendingAppointment.getStartDateTime());
     }
 
-    private List<Appointment> getPendingOccurrences(String appointmentUuid) {
+    private List<Appointment> getPendingOccurrences(String appointmentUuid, int highestStatusSequence) {
```

Comment:
Can we have appointmentStatus instead of highestStatusSequence which would be more clear?